extensions/clusters/kubernetesversions/all.go

```diff
@@ -236,14 +215,9 @@ func sortReleases(rancherVersion *semver.Version, releases []interface{}) (allAv
 			continue
 		}
 
-		maxVersion := release.(map[string]interface{})["maxChannelServerVersion"].(string)
 		minVersion := release.(map[string]interface{})["minChannelServerVersion"].(string)
```

Comment:
Have a qq, should we be removing the min version also from the code? Since max and min were both used to find the appropriate k8s versions?

Reply:
min version might still be necessary to generate a map of available versions, as Kubernetes versions vary based on specific minimum Rancher versions.

@caliskanugur Do you have inputs on this?

Reply:
Great question, it should affect it, but it might've also changed like this max version, we would need to check the newest response. 

I'd vote to update this logic one step at a time with a backlog. If any changes introduce some unexpected output, we can have a clearer look at the history of this sorting function to make minimal fixes. What do you all think?

Reply:
https://github.com/rancher/qa-tasks/issues/1123#issuecomment-1976989360, added this to an existing related ticket, awesome question @anupama2501

Reply:
Just realized that the min ver check was already removed here. So removing other rancher `minVersion`related things from this function should be safe

Reply:
Resolving this, we will stick with this approach as it's going to be used for the mapping